skiko/src/awtMain/kotlin/org/jetbrains/skiko/SkiaLayer.awt.kt

```diff
@@ -307,6 +316,7 @@ actual open class SkiaLayer internal constructor(
         }
         super.setBounds(x, y, roundedWidth, roundedHeight)
         backedLayer.setSize(roundedWidth, roundedHeight)
+        vrrDisplayLocker?.setBounds(0, 0, 1, 1)
```

Comment:
We need to call it after
```
vrrDisplayLocker = JPanel().apply {
 background = Color(0, 0, 0, 0)
}
```
too, because `setBounds` can be called before it

Reply:
probably we need to call it only there, and here we can remove `setBounds `

Reply:
Done.